.buildkite/pipeline.yml

```diff
@@ -835,6 +835,20 @@ steps:
       queue: central
       slurm_ntasks: 1
 
+    - label: ":computer: stretched 2D plane inertial gravity wave"
+    key: "cpu_inertial_gravity_wave"
```

Comment:
```suggestion
    key: "cpu_stretch_inertial_gravity_wave"
```

Reply:
Yup. Thanks! I figured

Reply:
@charleskawczynski I updated the key and the output directory (so that the run artifacts wouldn't be overwritten), CI is still complaining about not finding the expected key. Do you know why? :/

Reply:
Oh, I see! Silly unnecessary tab had slipped through when copied and pasted. Now it's fixed!